modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -6,6 +6,47 @@ endif::[]
 This topic describes common errors and steps to resolve them when installing Anypoint Runtime
 Fabric.
 
+== Troubleshoot Network Connectivity Using rtfctl
+
+Every Anypoint Runtime Fabric cluster requires connectivity with Anypoint control plane, and any interference with connectivity can limit functionality, resulting in application deployment failures or degraded status in Anypoint Runtime Manager.
+
+=== Obtain a Full Assessment
+Run the following command for an overall health assessment of the network:
+```
+/opt/anypoint/runtimefabric/rtfctl status
+```
+
+You can use `rtfctl` to verify that Runtime Fabric has the required outbound connectivity as well as troubleshoot connectivity issues. 
+
+=== Verify Outbound Connectivity
+
+On each node, follow the instructions in xref:install-rtfctl.adoc[Install rtfctl] to install rtfctl. 
+Run the following command in all controller and worker nodes on the cluster to verify the required outbound connectivity:
+```
+rtfctl test outbound-network
+```
+
+Sample output:
+```
+[root@rtf-controller-1 runtimefabric]# ./rtfctl test outbound-network
+Using proxy configuration from Runtime Fabric (proxy "", no proxy "")
+
+Using 'US' region
+transport-layer.prod.cloudhub.io:443 ✔
+https://anypoint.mulesoft.com ✔
+https://worker-cloud-helm-prod.s3.amazonaws.com ✔
+https://exchange2-asset-manager-kprod.s3.amazonaws.com ✔
+https://ecr.us-east-1.amazonaws.com ✔
+https://494141260463.dkr.ecr.us-east-1.amazonaws.com ✔
+https://prod-us-east-1-starport-layer-bucket.s3.amazonaws.com ✔
+https://runtime-fabric.s3.amazonaws.com ✔
+tcp://dias-ingestor-nginx.prod.cloudhub.io:5044 ✔
+```
+ 
+If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have the required whitelisting completed as described in xref:runtime-fabric::install-port-reqs#port-ips-and-hostnames-to-whitelist.
+ 
+For assistance using a TCP dump, refer to: “How to Capture Network Traffic Between Two Systems” (need a link to this KB article).
```

Comment:
Let's remove this line. We may decide to bring in the content from the support KB at a later time.

Reply:
Got it - thank you!